src/sentry/static/sentry/app/components/group/externalIssueForm.tsx

```diff
@@ -33,7 +33,7 @@ type Props = {
   group: Group;
   integration: Integration;
   action: 'create' | 'link';
-  onSubmitSuccess: (externalIssue: PlatformExternalIssue) => void;
```

Comment:
This data type was not the same as what is returned from the API. 

So I replaced it with `IntegrationExternalIssue`, which is what we use in other components.

Reply:
@NisanthanNanthakumar good catch, this is the right change.